docs/resources/rds_instance.md

```diff
@@ -293,8 +293,9 @@ The `volume` block supports:
 
 The `backup_strategy` block supports:
 
-* `keep_days` - (Optional, Int) Specifies the retention days for specific backup files. The value range is from 0 to
-  732. If this parameter is not specified or set to 0, the automated backup policy is disabled.
+* `keep_days` - (Optional, Int) Specifies the retention days for specific backup files. The value range is from 0 to 732.
+  If this parameter is not specified or set to 0, the automated backup policy is disabled.
```

Comment:
Is it conflict here by `If this parameter is not specified the automated backup policy is disabled` and `The default value is 7`?

Reply:
I have already edited it